packages/@o3r/schematics/src/utility/dependencies.ts

```diff
@@ -8,9 +9,14 @@ import { NodeDependency, NodeDependencyType } from '@schematics/angular/utility/
  *  @returns The version range value retrieved from the provided package.json file
  */
 export function getExternalDependenciesVersionRange<T extends string>(packageNames: T[], packageJsonPath: string): Record<T, string> {
-  const packageJsonContent = JSON.parse(fs.readFileSync(packageJsonPath, {encoding: 'utf-8'}));
+  const packageJsonContent = JSON.parse(fs.readFileSync(packageJsonPath, {encoding: 'utf-8'})) as PackageJson & { generatorDependencies: Record<string, string> };
   return packageNames.reduce((acc: Partial<Record<T, string>>, packageName) => {
-    acc[packageName] = packageJsonContent.peerDependencies?.[packageName] || packageJsonContent.generatorDependencies?.[packageName] || packageJsonContent.dependencies?.[packageName] || 'latest';
+    acc[packageName] =
+      packageJsonContent.peerDependencies?.[packageName] ||
+      packageJsonContent.generatorDependencies?.[packageName] ||
+      packageJsonContent.dependencies?.[packageName] ||
+      packageJsonContent.devDependencies?.[packageName] ||
+      'latest';
```

Comment:
should we keep that or throw an error ?

Reply:
That's a good question. We might consider logging a warning and fallbacking on `latest` so that we don't break the flow.

Reply:
I was thinking an error would be better to catch the exception directly in the it-tests, but I'm ok with the warning